app/classifier/annotation-renderer/svg.jsx

```diff
@@ -30,8 +30,10 @@ export default class SVGRenderer extends React.Component {
 
   getScale() {
     const sizeRect = this.getSizeRect();
-    const horizontal = sizeRect ? sizeRect.width / this.props.naturalWidth : 0;
```

Comment:
What does `0.001` mean? That sounds like it'll make everything tiny, wouldn't `1` be a more natural default scale factor?

Reply:
Setting a default of 1 means everything starts really big, then shrinks down to size once the image has loaded and the scale is set to, say, 0.5. This gives a noticeable jump in the size of the drawn markers, between the initial render and values being set, so I opted for a minimum value of `0.001` instead.

Reply:
That said, I am seeing an annoying pause in Firefox, sometimes, between the SVG image rendering and the drawn marks being rendered.